### PR TITLE
CREDENTIAL: Unified security error conditions

### PR DESCRIPTION
**Pull request by jmajnert**
Small cleanup. Hopefully will make the text clearer

### CODE REVIEW

File: specs/credentialmanagement/index.src.html [hunk 1]
```diff
@@ -747,20 +747,24 @@ <h3 id="request-credential">
...
+        </li>
+        <li>
+          <var>settings</var>' <a>responsible document</a> is not the
+          <a>active document</a> in the <a>top-level browsing context</a>
+        </li>
+        <li>
+          <var>settings</var> is not a <a>privileged context</a>
+        </li>
+        <li>
+          <var>environment</var> is not a <a>top-level browsing context</a>
```
Comment (mikewest) [inline specs/credentialmanagement/index.src.html]: Drop this one, please. It's covered by the 'responsible document' check.

File: specs/credentialmanagement/index.src.html [hunk 2]
```diff
@@ -747,20 +747,24 @@ <h3 id="request-credential">
...
     <li>
-      If <var>settings</var> does not have a <a>responsible document</a>, then
-      reject <var>promise</var> with a <code>SecurityError</code>
-      <strong>and</strong> terminate this algorithm.
-    </li>
-    <li>
-      If <var>settings</var>' <a>responsible document</a> is not the <a>active
-      document</a> in the <a>top-level browsing context</a>
-      If <var>environment</var> is not a <a>top-level browsing context</a>,
+      If any of the following conditions are met:
```
Comment (mikewest) [inline specs/credentialmanagement/index.src.html]: Please restructure this as `Reject <var>promise</var> ... if any of the following conditions are met:`. IMO it reads more clearly if it's not broken up in the middle.

File: specs/credentialmanagement/index.src.html (hunk 1 above)
Comment (mikewest) [inline specs/credentialmanagement/index.src.html]: And, I know you just added in a previous patch, but please drop the `environment` definition as well. We don't need it, as the "incumbent settings object" is the settings object we're referring to with `environment`.

**Comment by jmajnert** [comment]
Commit amended

**Comment by mikewest** [comment]
Next time, please just add another commit rather than squashing and rebasing. That makes it easier to track changes. :)

Otherwise, LGTM. Thanks!

**Pull request merged**